packages/server-wallet/benchmarks/fast-signature.benchmark.ts

```diff
@@ -24,10 +26,18 @@ async function benchmark(): Promise<void> {
   iter.map(() => signState(state, wallet.privateKey));
   console.timeEnd('signState');
 
+  const stateWithHash = addHash(state);
+
   console.time('fastSignState');
-  const result = iter.map(async () => fastSignState(state, wallet.privateKey));
+  const result = iter.map(async () => fastSignState(stateWithHash, wallet.privateKey));
```

Comment:
This would make the benchmark a fair comparison. (If done, you should delete `stateWithHash`, but the GH UI did not let me suggest that.)

```suggestion
  const result = iter.map(async () => fastSignState(addHash(state), wallet.privateKey));
```